### PR TITLE
[PE-3843] Fixed icon button accessibility issue

### PR DESCRIPTION
**Pull request by janibolkvadze**
### Commit summary

- Used `aria-label=""` approach
- Added support of `description=""` attribute for button CE (to be able to define CE Button aria-label from outside. Hardcoding of `aria-label='Icon'` inside of the CE would do the trick to pass accessibility tests, but all the buttons would have the same naming, so it wouldn't actually be accessible). Thoughts about attribute naming? @mattnickles @jllr 
- Updated button component documentation page
- Updated button CE documentation page
- Updated icon button tests

---
BTW. I suggest simplifying icon button CE by adding `icon=""` attribute.
Current markup uses `<chi-icon icon='atom'>` inside of `<chi-button type='icon'>`
Could be simply `<chi-button icon='atom'>`

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/419/ for more details.

File: src/custom-elements/src/components/button/button.tsx [hunk 1]
```diff
@@ -48,6 +48,11 @@ export class Button {
    */
   @Prop({ reflect: true }) center = false;
 
+  /**
+   *  to describe button for screen readers.
+   */
+  @Prop({ reflect: true }) description: string;
```
Comment (mattnickles) [inline src/custom-elements/src/components/button/button.tsx]: @janibolkvadze Let's change the attribute name from `description` to `alternative-text`

Comment (jllr) [inline src/custom-elements/src/components/button/button.tsx]: @janibolkvadze please change the att to alternative-text

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/436/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/437/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/438/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/439/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/441/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/442/ for more details.

**Pull request merged**